storm-client/src/jvm/org/apache/storm/Constants.java

```diff
@@ -30,7 +30,7 @@
     public static final List<Long> SYSTEM_EXECUTOR_ID = Arrays.asList(-1L, -1L);
     public static final String SYSTEM_COMPONENT_ID = "__system";
     public static final String SYSTEM_TICK_STREAM_ID = "__tick";
-    public static final String METRICS_COMPONENT_ID_PREFIX = "__metrics";
+    public static final String METRICS_COMPONENT_ID_PREFIX = "__metrics_";
```

Comment:
This is a minor change, but it might impact some people.  We probably want to just put this into 2.x for now.  If we really want it in 1.x we can decide how much of an impact it is (probably minor, but I want to see what others thing about it).

Reply:
Yea... But I made this change because https://github.com/apache/storm/blob/master/docs/Metrics.md says, 
```
The parallelism for each Bolt is set to parallelism.hint and component id for that Bolt is set to __metrics_<metrics consumer class name>
```

Reply:
That is fine, I think this is a really good fix.  I am just saying that it might make it difficult for someone to find say the LoggingMetricsConsumer in an automated way, and so if we do make this change in 1.x we should do it thoughtfully.

Reply:
Yes sure. Thanks for the review.